README.md

```diff
@@ -1,2 +1,119 @@
-# commit.dev-proto
-Marketing website for Commit (commit.dev)
+# Introduction
+Marketing website for Commit (commit.dev).
+
+You can find our staging website [here](https://landing.commit-staging.dev/).
+# How to Contribute
```

Comment:
Perhaps you can link this out to the wiki with some simple instructions that you have below (create changes on separate branch, and create a PR for reviewers to review, etc) 😄

Reply:
I will try and work on this afterwards.